### ISSUE TITLE
hh:mm:ss format in secondary axis

### ISSUE THREAD

**Issue by RubMoya** [AnyChart/AnyChart-Qlik#17]
Hi! First of all thanks a lot for the new version uploaded.

I am representing de data of a call center in a combo chart. I need to represent total number of calls by agent in a bar chart, then, in the secondary axis and using a line chart, the average duration (hh:mm:ss) of those calls by agent.

I have already used the secondary axis adding new scales but I haven't been able to figure out how to use the scale type "DateTime". Could you give me a hand, please?

Thanks for your time!

**Comment by RubMoya** [comment]
Let me add some files to clarify the issue:

I don't know if the format should be defined in the labels option.

![imagen](https://user-images.githubusercontent.com/42176025/44901280-3a57a980-ad07-11e8-8e08-10c1eaee5cd4.png)

![imagen](https://user-images.githubusercontent.com/42176025/44901313-53605a80-ad07-11e8-80ce-0ba29a6039d1.png)

**Comment by Shestac92** [comment]
@RubMoya 
This is possible!
First of all, the time value should be represented in milliseconds. For example, 210000 ms is equal to 00:03:30.
You are right, then you should create additional dateTime scale
![additionalscale](https://user-images.githubusercontent.com/24609062/44965500-8d7c6700-af68-11e8-9e1d-240bf745b2df.png)

Then you should bind axis to this scale
![yaxissettings1](https://user-images.githubusercontent.com/24609062/44965518-9d944680-af68-11e8-800e-ec63f1a3148d.png)

Then you should define 'HH:mm:ss' dateTime pattern in axis labels settings. Please, use the following string token for this - `{%tickValue}{dateTimeFormat: HH:mm:ss}`
![yaxissettings2](https://user-images.githubusercontent.com/24609062/44965603-0a0f4580-af69-11e8-9ea0-c6e2c38ef067.png)

And at the last step, you should bind the series to the additional scale
![bindingseries](https://user-images.githubusercontent.com/24609062/44965619-1d221580-af69-11e8-9dba-6b260266b6e4.png)

Here is a result
![result](https://user-images.githubusercontent.com/24609062/44965624-257a5080-af69-11e8-9a57-33dbb20d355a.png)

**Comment by RubMoya** [comment]
Hi @Shestac92 

Thanks a lot for your reply, it worked perfectly. Just one question...

![imagen](https://user-images.githubusercontent.com/42176025/44985327-51ed9700-af80-11e8-9d61-6c58b41690ed.png)

When the user selects points in the data line, it appears in milliseconds (instead of hh:mm:ss) Is there anyway the format of that inofrmation could be changed?

Thank you so much

**Comment by Shestac92** [comment]
Yes! You can use absolutely the same approach to the tooltip's content. Use the following string token for tooltip's content:
`Value: {%value}{dateTimeFormat: HH:mm:ss}`

Here is the result:
![screen shot 2018-09-04 at 10 29 51](https://user-images.githubusercontent.com/24609062/45006889-ae5ac000-b02d-11e8-9664-e85adbb69ab7.png)

**Comment by RubMoya** [comment]
Hi @Shestac92 

Thanks a lot for all your tips and help, anychart works great. I have been able to use all the time formats you shared with me perfectly, however, when I have to show and huge amount of hours (hundreds or thousands) the format doesn't seem to show them correctly. Is it just a 24h format? I have tried something like {%value}{dateTimeFormat: dd:HH:mm:ss} (for days) or {%value}{dateTimeFormat: HHHH:mm:ss} (for large hours) but I have no clue what else to try.

I want also ask you if there is any manual where I could find information about all the possible formats in AnyChart.

Once again, thanks a lot for your time !

**Comment by Shestac92** [comment]
@RubMoya 
As I can see from your message your data goes beyond 24 hours, right?

**Comment by RubMoya** [comment]
@Shestac92 yes. i am trying to represent the total amount of hours worked by an employee, so if i chose a range that include just a day it works fine, but when i select few months the amount of hours can be higher than 100 or even 1000. Thanks!

**Comment by RubMoya** [comment]
hI @Shestac92 ,
I was wondering if there is any expression to be able to show more than 24h. In some programms you need to use [hh]:mm:ss but it doesn't seem to work in Anychart.

Thanks a lot for your time.

**Comment by Shestac92** [comment]
@RubMoya 
Unfortunately, this approach is not available with the current version of AnyChart. Using string tokens the hh:mm format always show 12 or 24 hours based format. It can't show more than 24 hours.

**Comment by RubMoya** [comment]
hi there @Shestac92 , I was wondering if there is any way to represent duration greater than 24h in Anychart. Thanks in advanced!

**Comment by Shestac92** [comment]
@RubMoya 
Unfortunately, the current version of the AnyChart Qlik Extension doesn't provide this possibility. But I think we can look into it. I will suggest this improvement, we will discuss it and I will update you as I get results. Thank you for your suggestion!

**Comment by RubMoya** [comment]
@Shestac92 Thanks a lot for your answer.It would be very interesting to be able to show values greater than 24h. I will keep an eye on the updates.
